drools-compiler/src/main/java/org/drools/compiler/kie/builder/impl/BuildContext.java

```diff
@@ -33,4 +36,12 @@ public ResultsImpl getMessages() {
     public boolean registerResourceToBuild(String kBaseName, String resource) {
         return true;
     }
+
+    public void addIncludeModule(String kBaseName, InternalKieModule includeModule) {
+        // no op
+    }
+
+    public Map<String, InternalKieModule> getIncludeModules() {
+        return new HashMap<>();
```

Comment:
It would better to return `Collections.emptyMap()` here instead of creating a new map at each invocation.

Reply:
Thanks! Fixed in https://github.com/kiegroup/drools/pull/59